lib/slack-ruby-bot/rspec/support/fixtures/slack/migration_in_progress.yml

```diff
@@ -28,3 +28,31 @@ http_interactions:
       string: '{"ok":false,"error":"unknown"}'
     http_version:
   recorded_at: Tue, 28 Apr 2015 12:55:22 GMT
+- request:
+    method: post
+    uri: https://slack.com/api/rtm.start
+    body:
+      string: token=token
+  response:
+    status:
+      code: 200
+      message: OK
+    body:
+      encoding: UTF-8
+      string: '{"ok":false,"error":"migration_in_progress"}'
+    http_version:
+  recorded_at: Tue, 28 Apr 2015 12:55:22 GMT
+- request:
```

Comment:
This one should not be necessary.

Reply:
If the **'{"ok":false,"error":"unknown"}'** responses are removed, the migration_in_progress spec fails. It's part of the expectations: 
```
1) SlackRubyBot::Server retries on rtm.start errors migration_in_progress
     Failure/Error:
       expect do
         subject.run
       end.to raise_error Slack::Web::Api::Error, 'unknown'
     
       expected Slack::Web::Api::Errors::SlackError with "unknown" got #<VCR::Errors::UnhandledHTTPRequestError: 
```

Reply:
Oh I see. It's all good.